### PR TITLE
Orange dao community data(required web3py v6 support PR)

### PR DESCRIPTION
**Pull request by Andrei-Dolgolev**
## Changes

Add orange dao commnity data generation workflow.
Required moonworm PR https://github.com/bugout-dev/moonworm/pull/106

## How to test these changes?

Create queries:
```
custom-crawler queries --moonstream-token $MOONSTREAM_ORANGE_DAO_QUERIES_DATA_ACCESS_TOKEN init-tokenonomics --project orangedao
```

Approve them in queries journal.

Run data generation
```
custom-crawler orangedao --moonstream-token $MOONSTREAM_ORANGE_DAO_QUERIES_DATA_ACCESS_TOKEN run_tokenomics_orange_dao
```

Test commands:

Moonworm crawler
```
moonworm-crawler  --access-id "${NB_CONTROLLER_ACCESS_ID}" crawl -b ethereum
```

### CODE REVIEW

File: crawlers/deploy/ethereum-orange-dao-reports-tokenonomics.service [hunk 1]
```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Runs custom crawler for orange dao tokenonomics
+After=network.target
+
+[Service]
+Type=oneshot
+WorkingDirectory=/home/ubuntu/moonstream/crawlers/mooncrawl
+EnvironmentFile=/home/ubuntu/moonstream-secrets/app.env
+ExecStart=/home/ubuntu/moonstream-env/bin/python -m mooncrawl.reports_crawler.cli orangedao --moonstream-token {$ORANGE_DAO_QUERIES_TOKEN} run_tokenomics_orange_dao
```
Comment (kompotkot) [inline crawlers/deploy/ethereum-orange-dao-reports-tokenonomics.service]: Please rename to more recognizable way `MOONSTREAM_ORANGE_DAO_QUERIES_DATA_ACCESS_TOKEN`

File: crawlers/mooncrawl/mooncrawl/reports_crawler/cli.py [hunk 2]
```diff
@@ -571,13 +579,143 @@ def generate_report_nft_dashboard_handler(args: argparse.Namespace):
...
+        description="List all queries",
+    ).set_defaults(func=list_user_queries_handler)
+
+    init_tokenonomics_parser = general_queries_subparsers.add_parser(
+        "init-tokenonomics",
+        help="Create all predifind query",
+        description="Create all predifind query",
+    )
+
+    init_tokenonomics_parser.add_argument(
```
Comment (kompotkot) [inline crawlers/mooncrawl/mooncrawl/reports_crawler/cli.py:708]: This flag not used in service systemd files but required

**Review by kompotkot — CHANGES_REQUESTED** [review]

File: crawlers/mooncrawl/mooncrawl/reports_crawler/cli.py (hunk 2 above)
Comment (Andrei-Dolgolev) [reply crawlers/mooncrawl/mooncrawl/reports_crawler/cli.py:708]: Not required to run as systemd

File: crawlers/deploy/polygon-cu-nft-dashboard.service [hunk 3]
```diff
@@ -6,6 +6,6 @@ After=network.target
 Type=oneshot
 WorkingDirectory=/home/ubuntu/moonstream/crawlers/mooncrawl
 EnvironmentFile=/home/ubuntu/moonstream-secrets/app.env
-ExecStart=/home/ubuntu/moonstream-env/bin/python -m mooncrawl.cu_reports_crawler.cli cu-reports --moonstream-token "${MOONSTREAM_PUBLIC_QUERIES_DATA_ACCESS_TOKEN}" generate-nft-dashboard
+ExecStart=/home/ubuntu/moonstream-env/bin/python -m mooncrawl.reports_crawler.cli cu-reports --moonstream-token "${MOONSTREAM_PUBLIC_QUERIES_DATA_ACCESS_TOKEN}" generate-nft-dashboard
```
[the remark this answers is not in the document]
Comment (Andrei-Dolgolev) [reply crawlers/deploy/polygon-cu-nft-dashboard.service:9]: I left cu-reports and orangedao as separate cli it right now not generalized so well.

File: crawlers/deploy/polygon-cu-reports-tokenonomics.service [hunk 4]
```diff
@@ -6,6 +6,6 @@ After=network.target
 Type=oneshot
 WorkingDirectory=/home/ubuntu/moonstream/crawlers/mooncrawl
 EnvironmentFile=/home/ubuntu/moonstream-secrets/app.env
-ExecStart=/home/ubuntu/moonstream-env/bin/python -m mooncrawl.cu_reports_crawler.cli cu-reports --moonstream-token "${MOONSTREAM_PUBLIC_QUERIES_DATA_ACCESS_TOKEN}" queries run-tokenonomics
+ExecStart=/home/ubuntu/moonstream-env/bin/python -m mooncrawl.reports_crawler.cli cu-reports --moonstream-token "${MOONSTREAM_PUBLIC_QUERIES_DATA_ACCESS_TOKEN}" queries run-tokenonomics
```
[the remark this answers is not in the document]
Comment (Andrei-Dolgolev) [reply crawlers/deploy/polygon-cu-reports-tokenonomics.service:9]: I left `cu-reports` and `orangedao` as separate cli it right now not generalized so well.

**Comment by kompotkot** [comment]
@bugout-dev check
- [ ] Create env var MOONSTREAM_ORANGE_DAO_QUERIES_DATA_ACCESS_TOKEN

**Review by kompotkot — APPROVED** [review]
lg

**Pull request closed without merging**